Makefile

```diff
@@ -64,6 +64,7 @@ check: python-env
 	${FIND} -name *.py -exec autopep8 -d --max-line-length 120  {} \; | (! grep . -q) || (echo "Code differs from autopep8's style" && false)
 	# Validate that all updates were committed
 	$(MAKE) update
+	$(MAKE) notice
```

Comment:
I was thinking the same. The advantage would be that we don't waste time finding the issue when CI runs. The disadvantage is that it's executed on every `make update`. But it seems that the script is very quick, so I'm going to change it.